iModelJsNodeAddon/api_package/ts/src/NativeLibrary.ts

```diff
@@ -1363,7 +1363,7 @@ export declare namespace IModelJsNative {
 
   class ECSchemaXmlContext {
     constructor();
-    public addSchemaPath(path: string): void;
+    public addSchemaPath(schemaPath: string): void;

```

Comment:
why change this?

Reply:
i did, just to be more explicit, its just a var rename in the types so im pretty sure it wouldnt break anything, but i can revert

Reply:
should be fine. I'll approve